pkg/render/node.go

```diff
@@ -944,7 +944,7 @@ func (c *nodeComponent) cniEnvvars() []v1.EnvVar {
 		},
 	}
 
-	envVars = append(envVars, c.cfg.K8sServiceEp.EnvVars()...)
+	envVars = append(envVars, c.cfg.K8sServiceEp.EnvVars(true, c.cr.KubernetesProvider)...)
```

Comment:
I think you need to rebase on master. I think this will need to change to c.cfg.Installation.KubernetesProvider.